### ISSUE TITLE
loss cannot drop 

### ISSUE THREAD

**Issue by Bangbanggetnithecar** [CaoWGG/Mobilenetv2-CenterNet#5]
I changed the backbone of centernet to mobilenet. the loss didn't drop. I trained 500 epoches on pascalVOC dataset. the hm_loss stoped at 2.6. the wh_loss stoped at 9.6 after about 350 epoches and didn't changed.
我用了您的代码运行了一下，loss始终降不到特别低，一开始我在默认设置下运行了一下，hm_loss，wh_loss分别在350个epoch时停在了2.6和9.6并且不再降了，mAP只有0.09，我尝试改了一下wh_weight 和hm_weight 以及学习率等参数，mAP也只升到了0.16。希望你有想法的话可以交流一下，我的qq是1106741252，感谢您出色的工作
the opt setting is listed below
`==> torch version: 1.4.0
==> cudnn version: 7605
==> Cmd:
['main.py', 'ctdet', '--dataset', 'pascal', '--exp_id', 'voc_mbv2_0919', '--save_all', '--arch', 'mbv2_10', '--head_conv', '24', '--lr', '5e-4', '--batch_size', '128', '--gpus', '0,1,2,3,4,5,6,7']
==> Opt:
  K: 100
  aggr_weight: 0.0
  agnostic_ex: False
  arch: mbv2_10
  aug_ddd: 0.5
  aug_rot: 0
  batch_size: 128
  cat_spec_wh: False
  center_thresh: 0.1
  chunk_sizes: [16, 16, 16, 16, 16, 16, 16, 16]
  data_dir: /home/xiashaobang/Documents/CenterNet/src/lib/../../data
  dataset: pascal
  debug: 0
  debug_dir: /home/xiashaobang/Documents/CenterNet/src/lib/../../exp/ctdet/voc_mbv2_0919/debug
  debugger_theme: white
  demo: 
  dense_hp: False
  dense_wh: False
  dep_weight: 1
  dim_weight: 1
  down_ratio: 4
  eval_oracle_dep: False
  eval_oracle_hm: False
  eval_oracle_hmhp: False
  eval_oracle_hp_offset: False
  eval_oracle_kps: False
  eval_oracle_offset: False
  eval_oracle_wh: False
  exp_dir: /home/xiashaobang/Documents/CenterNet/src/lib/../../exp/ctdet
  exp_id: voc_mbv2_0919
  fix_res: True
  flip: 0.5
  flip_test: False
  gpus: [0, 1, 2, 3, 4, 5, 6, 7]
  gpus_str: 0,1,2,3,4,5,6,7
  head_conv: 24
  heads: {'hm': 20, 'wh': 2, 'reg': 2}
  hide_data_time: False
  hm_hp: True
  hm_hp_weight: 1
  hm_weight: 1
  hp_weight: 1
  input_h: 512
  input_res: 512
  input_w: 512
  keep_res: False
  kitti_split: 3dop
  load_model: 
  lr: 0.0005
  lr_step: [100, 300]
  master_batch_size: 16
  mean: [[[0.485 0.456 0.406]]]
  metric: loss
  mse_loss: False
  nms: False
  no_color_aug: False
  norm_wh: False
  not_cuda_benchmark: False
  not_hm_hp: False
  not_prefetch_test: False
  not_rand_crop: False
  not_reg_bbox: False
  not_reg_hp_offset: False
  not_reg_offset: False
  num_classes: 20
  num_epochs: 500
  num_iters: -1
  num_stacks: 1
  num_workers: 4
  off_weight: 1
  output_h: 128
  output_res: 128
  output_w: 128
  pad: 31
  peak_thresh: 0.2
  print_iter: 0
  rect_mask: False
  reg_bbox: True
  reg_hp_offset: True
  reg_loss: l1
  reg_offset: True
  resume: False
  root_dir: /home/xiashaobang/Documents/CenterNet/src/lib/../..
  rot_weight: 1
  rotate: 0
  save_all: True
  save_dir: /home/xiashaobang/Documents/CenterNet/src/lib/../../exp/ctdet/voc_mbv2_0919
  scale: 0.4
  scores_thresh: 0.1
  seed: 317
  shift: 0.1
  std: [[[0.229 0.224 0.225]]]
  task: ctdet
  test: False
  test_scales: [1.0]
  trainval: False
  val_intervals: 50
  vis_thresh: 0.3
  wh_weight: 0.1
`
If you have any ideas please contect me at qq(1106741252). Thank you for your excellent work

**Comment by sean1090** [comment]
> 

Same issue. The loss is still high after training 320 epoches. It seems that light backbones can't fit this model net well . Is there anything advice on light net with bbox and pose detection task？Thanks!
